docs/source/pages/tutorials/use_verification_summary_attestation.rst

```diff
@@ -0,0 +1,159 @@
+
+.. toggle::
+
+    .. code-block:: prolog
+
+        #include "prelude.dl"
+
+        Policy("gdk_provenance_policy", component_id, "Policy for GDK builds") :-
+            check_passed(component_id, "mcn_provenance_expectation_1")
+
+        apply_policy_to("has-hosted-build", component_id) :-
+            is_component(component_id, purl),
+            match("^pkg:maven/io.micronaut/micronaut-core@.*$", purl).
+
+This policy makes sure the :ref:`mcn_provenance_expectation_1 <checks>` check, which verifies the content of the provenance file matches :ref:`CUE expectation <pages/using:Verifying provenance expectations in CUE language>`. You can find the template policy files for GDK builds below:
+
+* `Template CUE expectation <https://github.com/oracle/macaron/tree/main/src/macaron/resources/policies/gdk/expectation.cue.template>`_
+* `Template Datalog policy file <https://github.com/oracle/macaron/tree/main/src/macaron/resources/policies/gdk/policy.dl.template>`_
+
+The VSA also includes the list of subjects and their corresponding checksums that have been verified, the version of Macaron used, the timestamp of the verification, and the result of the verification.
+
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+Automatically check the artifact checksum and verification result
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To verify that the artifact checksum matches the subject listed in the VSA and that the verification process has passed, follow these steps:
+
+
+**Download the check_vsa.sh script:**
+
+.. code-block:: shell
+
+    curl -O https://raw.githubusercontent.com/oracle/macaron/main/scripts/release_scripts/check_vsa.sh
+
+**Make the script executable:**
+
+.. code-block:: shell
+
+    chmod +x check_vsa.sh
+
+**Run the script with the appropriate arguments:**
+
+Following our example, let’s verify that the VSA has passed for the artifact available at `<https://maven.oracle.com/public/io/micronaut/micronaut-core/4.6.3-oracle-00001/micronaut-core-4.6.3-oracle-00001.jar>`_. You can either download the JAR from the repository or, if you have built the GDK project, obtain the artifact from your local Maven repository at ``~/.m2/repository/io/micronaut/micronaut-core/4.6.3-oracle-00001/micronaut-core-4.6.3-oracle-00001.jar``. Then, run the following command:
+
+.. code-block:: shell
+
+    ./check_vsa.sh --artifact-path micronaut-core-4.6.3-oracle-00001.jar --vsa-path vsa.intoto.jsonl --purl "pkg:maven/io.micronaut/micronaut-core@4.6.3-oracle-00001?type=jar"
+
+The artifact and VSA paths should be valid paths on your filesystem. Ensure you replace ``micronaut-core-4.6.3-oracle-00001.jar``, ``vsa.intoto.jsonl``, and ``pkg:maven/io.micronaut/micronaut-core@4.6.3-oracle-00001?type=jar`` with your actual file paths and package URL.
+
+**Verify the output:**
+
+If the verification is successful, the script will print:
```

Comment:
```suggestion
If the verification is successful, the script will return status code ``0`` and print this message to stdout:
```

Reply:
Returning a status code `0` is the default behavior and does not need to be explicitly mentioned.